packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -259,7 +259,7 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
   }
 
   /// Draws the selectionOverlay.
-  Widget _buildSelectionOverlay(Widget selectionOverlay) {
+  Widget _buildSelectionOverlay(Widget? selectionOverlay) {
```

Comment:
nit: I think you can skip building the entire stack entry if `selectionOverlay` is null, in the `build` method.

Reply:
```dart

if (widget.selectionOverlay != null) {
  _buildSelectionOverlay(widget.selectionOverlay)
}

```

I think so too.